tests/reftests/opamroot-versions.test

```diff
@@ -2712,8 +2746,8 @@ compiler: ["i-am-sys-compiler.2"]
 installed: ["i-am-package.2" "i-am-sys-compiler.2"]
 opam-version: "2.0"
 roots: ["i-am-package.2" "i-am-sys-compiler.2"]
-### :V:6:e: reinit from 2.2~alpha
-### ocaml generate.ml 2.2~alpha
+### :V:6:e: reinit from 2.2~beta
```

Comment:
At each bump root, a new entry should be added, I updated the PR

Reply:
Thanks! That looks good to me